internal/pkg/externalplugins/blunderbuss/blunderbuss.go

```diff
@@ -15,14 +15,12 @@ import (
 	"k8s.io/test-infra/prow/pkg/layeredsets"
 	"k8s.io/test-infra/prow/pluginhelp"
 	"k8s.io/test-infra/prow/plugins"
-	"k8s.io/test-infra/prow/plugins/assign"
+	assign "k8s.io/test-infra/prow/plugins/assign"
```

Comment:
Why we need rename?

Reply:
Maybe the IDE modified it automatically, I will revert it.